eng/Packaging.targets

```diff
@@ -46,7 +46,7 @@
     <ItemGroup>
       <_PackageIdentityWithoutPrerelease Include="$(Id)">
         <Version>$(PackageVersion)</Version>
-        <BlockStable>$(BlockStable)</BlockStable>
+        <BlockStable>$(SuppressFinalPackageVersion)</BlockStable>
```

Comment:
@joperezr does this metadata needs to be called `BlockStable`? Where is it used?

Reply:
You can change it, it is only consumed in packages.builds.  I did this here: https://github.com/dotnet/corefx/pull/41448/commits/c8c6551310a253c7d470d3f5eaba5359e8f6a694

Reply:
Great, thanks 😄